.github/workflows/ci-debian.yml

```diff
@@ -19,7 +19,7 @@ jobs:
 
       - name: Initialize sources
         run: mkdir ${{ env.WORK_DIR }}; cd ${{ env.WORK_DIR }};
-             git clone -q --depth 1 -b main https://github.com/seapath/ci ci;
+             git clone -q --depth 1 -b paramvm https://github.com/seapath/ci ci;
```

Comment:
I do not understand, you just change the branch.

Reply:
I had to, as look as https://github.com/seapath/ci/pull/46 was not merged.
It now is so I will change this back to main.